sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0b1 (Unreleased)
+## 1.1.0b1 (2024-05-21)
```

Comment:
Two things:

1. Under Other Changes, can you change it to: This version and all future versions will require Python 3.8+. Python 3.7 is no longer supported.
2. Under Breaking Changes, I think it's fine to say that v1.0 is not supported, but let's say the service has changed to date-based API versioning and that the v1.0 service behavior is supported through the new version, 2024-05-01.